### ISSUE TITLE
Thread for main differential photometry calculations question.

### ISSUE THREAD

**Issue by dmikkili** [OSCAAR/OSCAAR#111]
After the main code does the differential photometry part, brett(@bmorris3) wanted me to make the fitting frame open up since that would be the next step for people working with actual data. After multiple attempts I realized that we can never know when the actual differential photometry subprocess finishes, because in the subprocess the plot is created, and until the user closes the plot the subprocess wont technically terminate. I then realized that is good, so that after the user closes the plot it will open the frame up. When i tried adding the code to put the frame to open in the run method from the `WorkerThread()` class after the subprocess finishes, nothing seems to happen. I cannot figure out why I cannot open a new frame from the run process in the thread class. I then modified the code so that in the runOscaar main method, instead of `worker = WorkerThread()`, i just put the following 3 lines, and it worked perfectly.

```
 diffPhotCall = "from oscaar import differentialPhotometry"
 subprocess.check_call(['python','-c',diffPhotCall])
 FittingFrame(self.outputTxt.GetValue())
```

So basically my question is why are you creating a thread here in the main runOscaar method, and if I change it so that we dont need threads anymore, will it drastically ruin the code in other parts? Or if you know why trying to create a new wx.frame from inside the run method of a thread class doesnt do anything, I would appreciate it.
